test/forms/ConfidentialInformationFormProviderSpec.scala

```diff
@@ -27,20 +28,11 @@ class ConfidentialInformationFormProviderSpec extends StringFieldBehaviours {
 
     val fieldName = "field1"
     val requiredKey = "confidentialInformation.error.field1.required"
-    val lengthKey = "confidentialInformation.error.field1.length"
-    val maxLength = 100
 
     behave like fieldThatBindsValidData(
       form,
       fieldName,
-      stringsWithMaxLength(maxLength)
-    )
-
-    behave like fieldWithMaxLength(
-      form,
-      fieldName,
-      maxLength = maxLength,
-      lengthError = FormError(fieldName, lengthKey, Seq(maxLength))
```

Comment:
Did we mean to delete these? They look like valid assertions to me

Reply:
`ConfidentialInformationFormProvider` does not have any requirement on the length 
So I removed the scenario on the text length

Reply:
Sure